pal03377/Athena-New
# Display structured grading instructions in Playground

### Motivation and Context



closes #155 

### Description


Display the SGIs correctly in the exercise details 

### Steps for Testing


1. Export the evaluation data (example data does not have SGIs I think) if not already done (after #154 was merged)
2. Open playground and select evaluation data
3. In request mode with any active module expand the disclosure for a selected exercise's details
4. Open grading instructions disclosure and check if it looks good
5. Repeat for other exercises to check if they also look good

### Screenshots


#### Exercise with grading instructions and SGIs
![localhost_3000_playground (5)](https://github.com/ls1intum/Athena/assets/5898705/5b01f671-3f8e-4517-a882-41078e9049ed)

#### Exercise with only SGIs 
![localhost_3000_playground (6)](https://github.com/ls1intum/Athena/assets/5898705/8140a558-4b12-491d-9009-58c1fa35d6bd)

#### Exercise with grading instructions and no SGIs
![localhost_3000_playground (3)](https://github.com/ls1intum/Athena/assets/5898705/54bfd506-6faf-425a-a129-0c27aec2dc24)

#### Exercise with no grading instructions and no SGIs
![localhost_3000_playground (4)](https://github.com/ls1intum/Athena/assets/5898705/d977540f-fd95-4ab2-8306-5ae47d8cd9c0)
